Assets/MRTK/SDK/Editor/Inspectors/UX/ObjectManipulator/ObjectManipulatorInspector.cs

```diff
@@ -188,6 +211,42 @@ public override void OnInspectorGUI()
                 EditorGUILayout.PropertyField(scaleLerpTime);
             }
 
+            EditorGUILayout.Space();
+            elasticsFoldout = EditorGUILayout.Foldout(elasticsFoldout, "Elastics", true);
+
+            if (elasticsFoldout)
+            {
+                // This two-way enum cast is required because EnumFlagsField does not play nicely with
+                // SerializedProperties and custom enum flags.
+                var newElasticTypesValue = EditorGUILayout.EnumFlagsField("Manipulation types using elastic feedback: ", (TransformFlags)elasticTypes.intValue);
+                elasticTypes.intValue = (int)(TransformFlags)newElasticTypesValue;
+
+                // If the particular elastic type is requested, we offer the user the ability
+                // to configure the elastic system.
+                TransformFlags currentFlags = (TransformFlags)elasticTypes.intValue;
```

Comment:
You might be seeing several of these strange Flags casts, this is due to how serialization of custom enums and custom inspectors don't play very nicely together.